clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheActiveEntity.java

```diff
@@ -204,7 +168,7 @@
    */
   // This method is intended for unit test use; modifications are likely needed for other (monitoring) purposes
   Set<String> getStores() {
-    return Collections.unmodifiableSet(new HashSet<String>(stores.keySet()));
+    return Collections.unmodifiableSet(new HashSet<String>(serverStoreManager.getStores().keySet()));
```

Comment:
You are right they don't belong here. Actually they will be moved to actual test cases of the service emerging out of #1207, and then these methods will be gone from entity.